examples/llm/src/models/param_init_fns.py

```diff
@@ -153,16 +176,46 @@ def generic_param_init_fn_(module, cfg, init_fn_):
             )
 
 
-def baseline_param_init_fn_(module, cfg):
-    init_fn_ = partial(torch.nn.init.normal_, mean=0.0, std=cfg.init_std)
+def _normal_init_(std, mean=0.0):
+    return partial(torch.nn.init.normal_, mean=mean, std=std)
+
+
+def _normal_param_init_fn_(module, cfg, std):
+    init_fn_ = _normal_init_(std=std)
 
     if cfg.get('verbose', 0) > 1:
         warnings.warn(
-            f'Using torch.nn.init.normal_ init fn mean=0.0, std={cfg.init_std}')
+            f'Using torch.nn.init.normal_ init fn mean=0.0, std={std}')
 
     generic_param_init_fn_(module, cfg, init_fn_)
 
 
+def baseline_param_init_fn_(module, cfg):
```

Comment:
since we out here, might as well allow `normal_` where we allow user to pass in both mean and std?
also enable `uniform_` if you're feeling up to it.

Reply:
so, the initialization fn will always be called `MODEL_INIT_REGISTRY[init_fn_name](module, self.cfg)`... so the only way to pass mean and std are through the cfg (i.e. not referencing other values in the cfg; `n_layers`)... so this request is equivalent to looking for `cfg.init_mean` (plus adding uniform, which I think should wait till someone wants it)?

Reply:
yeah mean would be passed as part of `cfg.init_mean`
we can leave it out till someone wants it.